CONTRIBUTING.md

```diff
@@ -223,7 +223,7 @@ You can run the formatter from the shell:
 ```bash
 cabal install fourmolu-0.4.0.0
 cd path/to/the/root/of/swarm/repo
-find src/ app/ test/ -name "*.hs" | xargs fourmolu --mode=inplace --cabal-default-extensions
```

Comment:
Lets have both here and mention that there is a script for convenience.

It’s only a oneliner and it does not change often.

Many shells and plugins make reusing previous commands easy, so it’s possible some developers (like me) will prefer the explicit command.